command/server.go

```diff
@@ -1131,15 +1131,6 @@ func (c *ServerCommand) Run(args []string) int {
 	if envLicense := os.Getenv(EnvVaultLicense); envLicense != "" {
 		config.License = envLicense
 	}
-	if disableSSC := os.Getenv(DisableSSCTokens); disableSSC != "" {
-		var err error
-		config.DisableSSCTokens, err = strconv.ParseBool(disableSSC)
```

Comment:
It seems that there is also a reference to the server config for DisableSSCTokens. Should that also be removed?

Reply:
From what I could see that server config isn't actually usable from the config file (`hcl:"-"`) so it shouldn't matter, right?  The jira advised not cleaning up the code at this time, so all I did was remove the ability for a user to set the toggle.

Reply:
True, but I am wondering why to keep that around if it is a dormant line of code. It makes sense though to keep the scope within what Jira is asking! Thanks for the context.

Reply:
Yeah, I was afraid if I started down that path I wouldn't stop :)